src/assets/validation.js

```diff
@@ -1,3 +1,6 @@
+const emailRegex = /^\w+(?:[\.-]\w+)*@(?:gmail\.com|yahoo\.com|hotmail\.com|aol\.com|outlook\.com)$/;
```

Comment:
![image](https://github.com/agamjotsingh18/pollitup/assets/96580571/df85c100-938d-403f-8e2f-3143ca6d9c4a)
you need to update the email regex currently it only accepts a small number of domain ... remove the requirement for a specific domain and let users use anything separated with dot.

Reply:
okay got it. 👍🏻